Pipfile

```diff
@@ -8,6 +8,7 @@ django = "*"
 
 [dev-packages]
 flake8 = "*"
+pytest-django = "*"
```

Comment:
Try to add the line: `pytest = "*"` in addition to `pytest-django = "*"`.
Also, I think maybe you need to re-create the Pipfile.lock as @Omeramsc mentioned before because you edit the file Pipfile.